### PR TITLE
Tentative XDM JSON properties

### PR DESCRIPTION
**Pull request by vladbailescu**

### CODE REVIEW

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java [hunk 1]
```diff
@@ -47,6 +49,7 @@ default String getId() {
      *
      * @since com.adobe.cq.wcm.core.components.models.datalayer 1.0.0
      */
+    @JsonProperty("repo")
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java:52]: Is `repo` the asset URL or the URL of the repository (store) where the asset is stored?

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ContainerDataModel.java [hunk 2]
```diff
@@ -29,6 +31,7 @@
      *
      * @since com.adobe.cq.wcm.core.components.models.datalayer 1.0.0
      */
+    @JsonProperty("xdm:items")
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ContainerDataModel.java:34]: IIUC `xdm:items` represents all the items of the container which is different from the displayed items.

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ImageDataModel.java [hunk 3]
```diff
@@ -31,7 +31,7 @@
      *
      * @since com.adobe.cq.wcm.core.components.models.datalayer 1.0.0
      */
-    @JsonProperty("asset")
+    @JsonProperty("image")
     default AssetDataModel getImage() {
```
Comment (jckautzmann) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ImageDataModel.java:35]: I'd call this method `getAssetData()`

Comment (vladbailescu) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ImageDataModel.java]: Done in 2d1846aedbd65f6c9483a948382dc9ee13d9588b

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java (hunk 1 above)
Comment (vladbailescu) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java]: That was supposed to be `repo:path` (https://github.com/adobe/xdm/blob/master/docs/reference/external/repo/asset.schema.md#repopath)

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ContainerDataModel.java (hunk 2 above)
Comment (vladbailescu) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ContainerDataModel.java]: You're right, we only list the active items. I will switch this back to our custom property name.

Comment (vladbailescu) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/ContainerDataModel.java]: Done in f71cfef

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java (hunk 1 above)
Comment (vladbailescu) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/datalayer/AssetDataModel.java]: Done in f71cfef

**Pull request merged**